### DISCUSSION TITLE
Bottom navigation bar reference

### DISCUSSION THREAD

**Discussion by NikunjK8** [fast4x/RiMusic#1358] [General]
A different fork of ViMusic called [Music You](https://github.com/DanielSevillano/music-you) has implemented bottom navigation bar. Maybe this could be used for reference to do the same in RiMusic?
(This app actually takes inspiration from RiMusic as well)

**Comment by fast4x** [comment]
Yes, i can check this fork

**Reply by outerowl** [reply]
I thing rimusic team should just give option to user set navigation menu according to their preference but side navigation bar makes rimusic unique ❤ and that should be the default navigation bar position.

**Reply by NikunjK8** [reply]
There's no team behind it. Only a single hard working person @fast4x. Also the app already has option in UI settings to change navigation bar position and left position is indeed default.

**Reply by outerowl** [reply]
> There's no team behind it. Only a single hard working person @fast4x. Also the app already has option in UI settings to change navigation bar position and left position is indeed default.

Thanks for informing I was not aware to this thing @fast4x Great work love from Bihar ❤.
